Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocValidationErrorIndicatorColumnRenderer.cs

```diff
@@ -65,6 +65,33 @@ protected override string GetAdditionalCssClassForDataCell (BocCellAttributeRend
       return CssClasses.DataCellValidationFailureIndicator;
     }
 
+    protected override void RenderTitleCellContents (BocColumnRenderingContext<BocValidationErrorIndicatorColumnDefinition> renderingContext)
+    {
+      var columnTitle = renderingContext.ColumnDefinition.ColumnTitle;
+      if (columnTitle.IsEmpty)
+        columnTitle = WebString.CreateFromText("Validation errors");
+
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClasses.ValidationErrorMarker);
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Title, columnTitle.ToString(WebStringEncoding.Html));
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Span);
+
+      var iconUrl = renderingContext.Control.ValidationFailureRepository.HasValidationFailuresForDataRows()
+          ? c_validationIndicatorIcon
+          : c_validationIndicatorDisabledIcon;
```

Comment:
I looked into the represenative samples:
We have used
```
img {
    opacity: 0.5;
    filter: grayscale(100%);
}
```
for disabled icons in NovaViso.

Alternative might be the gray-shade from the null-Checkbox value `#979797` in NovaViso

Disclaimer: need to see it live for final word.